build.sbt

```diff
@@ -365,6 +365,7 @@ lazy val reflect = configureAsSubproject(project)
   .settings(
     name := "scala-reflect",
     description := "Scala Reflection Library",
+    compileOrder := CompileOrder.Mixed,
```

Comment:
this is the default, no? so no need to set it.

Reply:
I think it is not, the build defaults to `JavaThenScala`.